windows/open_command_prompt.tw2

```diff
@@ -0,0 +1,46 @@
+::Opening the Command Prompt on Windows 10
+# Opening the Command Prompt on Windows 10
+
+Now that we have downloaded Exercism, we need to move it to the right folder. To do so, we'll use the Command Prompt.
+
+Let's get started by opening the Command Prompt.
+
+---
+
+## Step 1: Opening Start
+### Exercise
+
+Open the Start menu, by clicking the Windows icon in the bottom left corner of your screen.
+
+
+## Step 2: Searching for the Command Prompt
+### Exercise
+With the Start menu open, start typing `cmd.exe` to search for the Command Prompt
+
+### Verify
+
+When searching for `cmd.exe` in the Start menu, your results will look something like this:
+
+![cmd.exe result](https://transfer.sh/rcwkt/win10-cmd.png)
+
+
+## Step 3: Open the Command Prompt
+### Exercise
+Double click the top result, as shown above, to open the Command Prompt.
+
+### Verify
+
+After double clicking, you'll see a window like the one shown here:
+
+![cmd.exe window](https://transfer.sh/jnNbx/win10-cmd-result.png)
```

Comment:
I think it would make sense to store them centralised, instead of on an image sharing service - https://github.com/exercism/interactive-cli-walkthrough/issues/18 was opened today

Reply:
That's perfect, thanks!